composer.json

```diff
@@ -22,7 +22,8 @@
     "config": {
         "sort-packages": true,
         "allow-plugins": {
-            "dealerdirect/phpcodesniffer-composer-installer": true
+            "dealerdirect/phpcodesniffer-composer-installer": true,
+            "laminas/laminas-dependency-plugin": false
```

Comment:
Do you need this configuration?
`laminas/laminas-dependency-plugin` does not seem to be installed at all.